cms/static/cms/js/jquery.cms.wysiwyg.js

```diff
@@ -1,7 +1,9 @@
 django.jQuery(function($){
-    $('.wysiwyg:visible').each(function(){
-        activate_tinymce(this)
-    });
+    setTimeout(() => {
```

Comment:
Should this be `setInterval`?

Reply:
No? I'm just delaying it.

Reply:
How does it render WYSIWYG's that aren't in the viewport on initial page load?

Reply:
Hmm, it doesn't. That's an existing bug it seems - I didn't change the selector, just added a delay.

Reply:
Sorted in onespacemedia/project-template#237.